src/main/java/frc/robot/Limelight.java

```diff
@@ -1,17 +1,57 @@
+package frc.robot;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.geometry.Translation2d;
 import edu.wpi.first.networktables.NetworkTable;
-import edu.wpi.first.networktables.NetworkTableEntry;
 import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import static frc.robot.Constants.LimelightConstants.*;
+
 
 class Limelight {
 
+    /**
+     * Creates a network table instance and an entry for each necessary value
+     */
     private final NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
 
+    /**
+     * Creates a default array of values for use with the cam-tran table entry
+     */
+    private Number[] defaultValues =
+    {
+        0.0f,
+        0.0f,
+        0.0f,
+        0.0f,
+        0.0f,
+        0.0f
+    };
+
+    /**
+     * Creates a number Array to store the robot position values from the cam-tran entry
+     * Fills it with the numbers from the defaultValues array
+     */
+    private Number[] getRobotPosition3D()
+    {
+        Number[] robotPositionValues = table.getEntry("camtran").getNumberArray(defaultValues);
+        return robotPositionValues;
```

Comment:
can't find where this table is mentioned in the networktables api docs or anywhere in limelight docs. has this been replaced since last year?

Reply:
This has been deprecated very recently, and I have replaced it with the updated botPose array